File: app/src/main/java/com/example/assignment/views/LoginActivity.java
package com.example.assignment.views;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.assignment.R;
import com.example.assignment.databinding.ActivityLoginBinding;

public class LoginActivity extends AppCompatActivity {

    private ActivityLoginBinding activityLoginBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        activityLoginBinding = ActivityLoginBinding.inflate(getLayoutInflater());
        setContentView(activityLoginBinding.getRoot());
    }
}

File: app/src/main/java/com/example/assignment/views/LoginActivity.kt
package com.example.assignment.views

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.assignment.databinding.ActivityLoginBinding

class LoginActivity : AppCompatActivity() {
    private var activityLoginBinding: ActivityLoginBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        activityLoginBinding = ActivityLoginBinding.inflate(
            layoutInflater
        )
        setContentView(activityLoginBinding!!.root)
    }
}